src/monitor.js

```diff
@@ -77,6 +77,64 @@ class Monitor {
       newopts
     );
   }
+
+  // Given a function that operates on a
+  // single message, this will time it and
+  // report to statsum.
+  timedHandler(name, handler) {
+    return async (message) => {
+      let start = process.hrtime();
+      let success = 'success';
+      try {
+        await handler(message);
+      } catch (e) {
+        success = 'failure';
+        throw e;
+      } finally {
+        let d = process.hrtime(start);
+        this.measure(name + '.' + success, d[0] * 1000 + (d[1] / 1000000));
+        this.measure(name + '.all', d[0] * 1000 + (d[1] / 1000000));
+      }
+    }
+  }
+
+  // Given an express api method, this will time it
+  // and report to statsum.
+  expressMiddleware(method) {
```

Comment:
how about we call it `prefix` or `name`, to just stay consistent... but I agree this is the method name...